### DISCUSSION TITLE
USB A to C adapter

### DISCUSSION THREAD

**Discussion by improvius** [MoleMan1024/audiowagon#64] [General]
If anyone is interested, I am using an Anker USB-C Male to USB-A Female adapter AK-848061067804 with an older USB-A thumb drive in my XC40 Recharge, and it is working perfectly with Audiowagon. Link to adapter on Amazon: https://www.amazon.com/Anker-Adapter-Converts-Technology-Compatible/dp/B01COOQIKU

**Comment by floydp386** [comment]
I'm using one of these and also works perfectly 
Amazon - https://www.amazon.co.uk/Adapter-Converting-Thunderbolt-MacBook-Devices/dp/B07CVX3516/ref=sr_1_3?crid=21UIV4C69BLNC&keywords=USB+C+to+USB+Adapter%282+Pack%29%2C+Syntech&qid=1677081061&sprefix=usb+c+to+usb+adapter+2+pack+syntech+%2Caps%2C142&sr=8-3
